### PR TITLE
CNV-8966: New content to recommend SSH key use

### PR DESCRIPTION
**Pull request by sabrinajess**
For 4.8 and later
Addresses [CNV-8966](https://issues.redhat.com/browse/CNV-8966) and [CNV-10117](https://issues.redhat.com/browse/CNV-10117)

Relevant preview links:
- https://deploy-preview-33086--osdocs.netlify.app/openshift-enterprise/latest/virt/vm_templates/virt-creating-vm-template.html
- https://deploy-preview-33086--osdocs.netlify.app/openshift-enterprise/latest/virt/virtual_machines/virt-create-vms.html

### CODE REVIEW

**Comment by sabrinajess** [comment]
@yaacov and @stu-gott please review for CNV-8966 and CNV-10117, respectively.

**Comment by yaacov** [comment]
LGTM, @gouyang can you review for QE ?

**Review by yaacov — APPROVED** [review]

**Comment by gouyang** [comment]
LGTM.

**Comment by stu-gott** [comment]
LGTM

File: modules/virt-cloud-init-fields-web.adoc [hunk 1]
```diff
@@ -19,3 +19,8 @@
 |Custom script
 |Replaces other options with a field in which you paste a custom cloud-init script.
 |===
+
+[NOTE]
+====
+Statically inject an SSH key using the custom script in Cloud-init or in the wizard. This allows you to securely and remotely manage virtual machines and manage and transfer information. This step is strongly recommended to secure your VM.
```
Comment (ousleyp) [inline modules/virt-cloud-init-fields-web.adoc:25]: The note says "This step is strongly recommended," so I wonder if it actually belongs in [the procedure](https://deploy-preview-33086--osdocs.netlify.app/openshift-enterprise/latest/virt/vm_templates/virt-creating-vm-template.html#virt-creating-template-wizard-web_virt-creating-vm-template) under step 8.

Nitpick 1: I think it's generally "cloud-init" without the capital C, unless it's the beginning of a sentence. (If it's referring to a UI element titled "Cloud-init," the text should be bold.)

Nitpick 2: s/using the custom script/by using the custom script

File: modules/virt-creating-template-wizard-web.adoc [hunk 2]
```diff
@@ -41,7 +41,7 @@ A *Bootable Disk* is not required for virtual machines provisioned from a *PXE*
 Blank disks, PVC disks without a valid boot source, and the cloudinitdisk cannot be used as a boot source.
 ====
 
-. Optional: Click *Advanced* to configure Cloud-init.
+. Optional: Click *Advanced* to configure Cloud-init and SSH access.
```
Comment (ousleyp) [inline modules/virt-creating-template-wizard-web.adoc:44]: You didn't put this there, but s/Cloud-init/cloud-init

**Review by ousleyp — COMMENTED** [review]
Mostly this LGTM but I have a placement question and a couple nits :)

**Pull request merged**